### ISSUE TITLE
Extra space in call number string in SOS

### ISSUE THREAD

**Issue by winthropo** [NEU-Libraries/scholar-onesearch#138]
Could this extra space be removed? Low importance.

![screen shot 2014-09-05 at 10 44 46 am copy](https://cloud.githubusercontent.com/assets/4663357/4166446/69143da6-350b-11e4-87d6-263d0e9732a6.jpg)

**Comment by nestochan** [comment]
from the SOS issues basement: @winthropo @kflachs I think the normalization for this is thus: 

<img width="1015" alt="screenshot 2015-08-28 12 15 06" src="https://cloud.githubusercontent.com/assets/2441612/9551447/49b64192-4d7f-11e5-8730-b7006004c6b4.png">

I've deployed the norm change in staging  and issued a renormalization when it's done, we can see how it looks. :black_joker:

**Comment by kflachs** [comment]
Ernesto informed me on Friday afternoon that this did not work. I tried one other strategy today (Tuesday) and it did not work either.  Is this a normalization issue?  This assumes that there is an extra space after each date in the call number.  I've checked the Steve Jobs record and spot-checked others, and there isn't an extra space after the date; therefore there is nothing for Primo to remove when normalizing.  The space is being added somewhere else.  Could this be somewhere in the stylesheets or some place else?

**Comment by nestochan** [comment]
it's actually showing up in the pnx from what I can tell.  pehaps adding a space in front will balance the string.

On Tue, Sep 8, 2015 at 1:36 PM -0700, "kflachs" <notifications@github.com<mailto:notifications@github.com>> wrote:

Ernesto informed me on Friday afternoon that this did not work. I tried one other strategy today (Tuesday) and it did not work either. Is this a normalization issue? This assumes that there is an extra space after each date in the call number. I've checked the Steve Jobs record and spot-checked others, and there isn't an extra space after the date; therefore there is nothing for Primo to remove when normalizing. The space is being added somewhere else. Could this be somewhere in the stylesheets or some place else?

##

**Comment by kflachs** [comment]
I found a case for the added space in ExLibris.  (00134882).  When the normalization rule "add at end of string" is used, the way it is coded, it is apparently adding an extra space.  That's why the normalization rule can't remove the space since it's that very rule that is adding it in through the coding.

I've staged a workaround and am renorming.  We might know at the end of the day, if not, Thursday morning.
